.github/workflows/ci.yml

```diff
@@ -0,0 +1,94 @@
+name: Kubernetes-sigs/krew CI
+on: 
+  push:
+    branches: 
+      - '*'
+    tags: 
+      - 'v*.*.*'
+  pull_request:
+    branches: 
+      - '*'
+  release:
+    types: [published, created, edited]
+
+jobs:
+
+  build:
+    name: Build and Tags
```

Comment:
you can drop `name:` since there's only 1 job.

Reply:
Sure, np :)